src/arrayMethodSort.js

```diff
@@ -5,7 +5,35 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
```

Comment:
you can add here the default parametr
<img width="248" alt="image" src="https://github.com/mate-academy/js_array-method-sort/assets/70220667/7ad59b61-85e5-4d88-8034-35cd70a10014">

compareFunciton = defaultCompareFunction